core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/DistroXV1RequestToStackV4RequestConverter.java

```diff
@@ -7,6 +7,7 @@
 
 import javax.inject.Inject;
 
+import com.sequenceiq.common.api.type.InstanceGroupType;
```

Comment:
nit: This import is in the wrong place. It should be moved down with the rest of the `import com.*` below.